gax-grpc/src/test/java/com/google/longrunning/OperationsClientTest.java

```diff
@@ -236,4 +237,46 @@ public void deleteOperationExceptionTest() throws Exception {
       // Expected exception
     }
   }
+
+  @Test
+  @SuppressWarnings("all")
+  public void waitOperationTest() {
+    String name2 = "name2-1052831874";
+    boolean done = true;
```

Comment:
inline

Reply:
Would prefer to keep this as close to the auto-generated code as possible, so that manual, semantic-impacting changes can be easily identified if this LRO GAPIC ever needs to be regenerated.